src/ansys/fluent/core/launcher/error_handler.py

```diff
@@ -1,10 +1,26 @@
 """Provides a module for customized error handling."""
 
+import os
+
 from ansys.fluent.core.exceptions import InvalidArgument
 from ansys.fluent.core.launcher import launcher_utils
 from ansys.fluent.core.utils.fluent_version import FluentVersion
 
 
+class InaccessibleAnsysLicense(RuntimeError):
+    """Raised when license is inaccessible."""
+
+    def __init__(self):
+        super().__init__(
+            "Enable license server in `Licensing Settings` application and set `ANSYSLMD_LICENSE_FILE` environment variable correctly."
+        )
+
+
+def _check_license():
+    if not os.getenv("ANSYSLMD_LICENSE_FILE", None):
```

Comment:
@hpohekar I am able to use Fluent without the ANSYSLMD_LICENSE_FILE environment variable set

Reply:
@cj-hodgson I'm not able to launch Fluent using PyFluent without the ANSYSLMD_LICENSE_FILE environment variable set. It gives same error as mentioned in the issue description.

Reply:
@cj-hodgson Are you launching Fluent directly or using PyFluent ?

Reply:
@hpohekar In fact, we don't need to provide any advice in the message. The user has access to Ansys documentation, which we can leave as their concern. We can just tell them the reason for any failure.

Reply:
Yes, I agree.

Reply:
@hpohekar Both. I don't see licensing issues when I've setup licensing via Licensing Settings and don't have ANSYSLMD_LICENSE_FILE set

Reply:
@cj-hodgson  Ok. Thank you. We are handling this exception based on it's type and message now.